src/Servers/Kestrel/Transport.Quic/README.md

```diff
@@ -1,3 +1,167 @@
-For external contributors, msquic.dll isn't available publicly yet. See https://github.com/aspnet/Announcements/issues/393.
+# Getting start with HTTP/3 on Windows and Linux
 
-Credit to Diwakar Mantha and the Kaizala team for the MsQuic interop code.
\ No newline at end of file
+This is a getting started guide for using HTTP/3 on Windows and Linux.
+
+## Prerequisites
+
+1. Download the latest 5.0-preview build of .NET from <https://dotnet.microsoft.com/download/dotnet/5.0.>
+2. Download Edge Canary <https://www.microsoftedgeinsider.com/en-us/download.> This is required to browse websites with HTTP/3.
+
+### Windows
+
+1. Latest [Windows Insider Builds](https://insider.windows.com/en-us/), Insiders Fast build. This is required for Schannel support for QUIC.
```

Comment:
What's the min version required? Is 2004 sufficient? Even better if you can add a powershell one-liner that checks if the OS is new enough